packages/ts/react-crud/src/header-filter.tsx

```diff
@@ -82,33 +88,67 @@ function ComparationSelection({ onMatcherChanged, value }: ComparationSelectionP
 }
 
 export function StringHeaderFilter(): ReactElement {
+  const context = useContext(ColumnContext)!;
+  const { filterPlaceholder, filterDebounceTime, filterMinLength } = context.customColumnOptions ?? {};
   const { updateFilter } = useFilterState(Matcher.CONTAINS);
+  const [inputValue, setInputValue] = useState('');
+
+  useEffect(() => {
+    if (filterMinLength && inputValue && inputValue.length < filterMinLength) {
```

Comment:
The behavior here could be confusing: It only starts filtering after entering the min. number of characters. However after removing a single character, it does not remove the filter and still shows the filter result. Maybe it should reset the filter immediately if the filter value has less than the minimum number of characters? Not quite sure which is better.

Reply:
Agreed, and made changes that the filter is cleared with empty input value if input value length is less than min length